docs/networking.md

```diff
@@ -0,0 +1,175 @@
+# How to use networking
+
+cloud-hypervisor can emulate one or more virtual network interfaces, represented at the hypervisor host by [tap devices](https://www.kernel.org/doc/Documentation/networking/tuntap.txt"). This guide briefly describes how to setup and use networking with cloud-hypevisor.
+
+## Start cloud-hypervisor with net devices
+
+Use one or more `--net` command-line argument from __cloud-hypervisor__ to specify the emulation of one or more virtual NIC's. The example below instructs cloud-hypervisor to emulate for instance 2 virtual NIC's:
```

Comment:
Should we have a simpler setup with one single interface only? And then have a final section briefly describing how to extend it to multiple interfaces?
I feel that the single interface+bridge+NAT/IP forward is the most common setup.

Reply:
It depends I think. Again for quick testing scenarios this holds. For cloud deployments perhaps even based on SDN with DPDK openvswitch it might be different though. Also when running multiple CH's an integration bridge for OAM to all instances is quite common. If you prefer your proposal then pls go ahead and I will delete this PR.

Reply:
> If you prefer your proposal then pls go ahead and I will delete this PR.

No, please keep this PR and we'll go with 2 interfaces as an example.